### ISSUE TITLE
p-autoComplete Accessibility Issues

### ISSUE THREAD

**Issue by geromegrignon** [openng-org/optimus-ui#348]
### Describe the bug

The p-toggleSwitch component with checkboxes has accessibility violations when running the IBM Equal Accessibility Checker. The issue is: The element with role "option" contains descendants with roles "combobox" which are ignored by browsers

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/stackblitz-starters-efdszjqx

### Environment

Stackblitz

### Angular version

20.0.4

### PrimeNG version

v20

### Steps to reproduce the behavior

1. Install the [IBM Equal Accessibility Checker](https://chromewebstore.google.com/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp)
2. Open the stackblitz
3. Run the accessibility checker
4. Filter on violations only and note the errors: 'The element with role "option" contains descendants with roles "combobox" which are ignored by browsers'

### Expected behavior

There are no accessibility violations for the p-autoComplete component

> Migrated from `primefaces/primeng#18769` — originally by `@srujana10499` on 2025-08-16

**Comment by geromegrignon** [comment]
*[Originally posted by `@github-actions[bot]` on 2026-02-13T00:29:45.000Z]*

This issue has been automatically marked as stale due to a lack of activity. **If this issue is still relevant to you, please leave a comment** so we can keep it open. We apologize for not being able to prioritize it sooner. If you have any new information or questions, please share them in your comment!
